Makefile should not write into `/usr/bin` as build step

```
lib % make
gcc -g -Wall -W -O2   -c -o lstbt.o lstbt.c
gcc -g -Wall -W -O2   -c -o lstbt_t.o lstbt_t.c
gcc -g -Wall -W -O2   -c -o lstbt_r.o lstbt_r.c
gcc -g -Wall -W -O2   -c -o lstbt_v.o lstbt_v.c
gcc -g -Wall -W -O2   -c -o router.o router.c
gcc -g -Wall -W -O2   -c -o adapter.o adapter.c
gcc -g -Wall -W -O2   -c -o helpers.o helpers.c
gcc -g -Wall -W -O2   -c -o ../utils.o ../utils.c
gcc -g -Wall -W -O2 -o /usr/bin/lstbt lstbt.o lstbt_t.o lstbt_r.o lstbt_v.o router.o adapter.o helpers.o ../utils.o -lm
/usr/lib/gcc/aarch64-unknown-linux-gnu/11/../../../../aarch64-unknown-linux-gnu/bin/ld: cannot open output file /usr/bin/lstbt: Permission denied
collect2: error: ld returned 1 exit status
make: *** [Makefile:25: /usr/bin/lstbt] Error 1
```
Prevents building `lstbt` as user without write access to `/usr/bin` and is surprising that a plain make would modify any files outside of the build or source directory (by default).

--- a/README.md
+++ b/README.md
@@ -14,14 +14,16 @@ lstbt is present in the ./lib subdirectory of the software.
 Steps:<br>
 1. `cd ./lib`
 2. `make`
+3. `make install`
 
-## Uninstalling lstbt and cleaning up the build
+## cleaning up the build
 
 Steps:<br>
 1. `cd ./lib`
 2. `make clean`
 
 Note: The library installs itself in the /usr/bin filesystem path, hence pertinent permissions are required for the user to alter it.
+Note 2: It is not necessary to install lstbt for testing it.
 
 ## TBT/USB4 user-space functionalities
 This software serves as the first prefatory abstraction of various functionalities of the TBT/USB4 subsystem a user can

--- a/lib/Makefile
+++ b/lib/Makefile
@@ -27,3 +27,4 @@ $(LIBTBT_EXEC): $(O_FILES)
 clean:
 	-$(RM) $(LIBTBT_EXEC) $(O_FILES)
 
+.PHONY: all clean

--- a/lib/Makefile
+++ b/lib/Makefile
@@ -5,9 +5,12 @@
 # Copyright (C) 2023 Rajat Khandelwal <rajat.khandelwal@intel.com>
 # Copyright (C) 2023 Intel Corporation
 
-LIBTBT_EXEC = /usr/bin/lstbt
+INSTALL_PATH=/usr/bin
+
+LIBTBT_EXEC = lstbt
 
 CC = gcc
+INSTALL = install
 RM = rm -f
 
 DEBUG_FLAGS = -g
@@ -24,7 +27,14 @@ all: $(LIBTBT_EXEC)
 $(LIBTBT_EXEC): $(O_FILES)
 	$(CC) $(CFLAGS) -o $@ $^ -lm
 
+install: $(LIBTBT_EXEC)
+	$(INSTALL) -d -m 755 $(DESTDIR)$(INSTALL_PATH)
+	$(INSTALL) $(LIBTBT_EXEC) $(DESTDIR)$(INSTALL_PATH)
+
+uninstall:
+	$(RM) $(DESTDIR)$(INSTALL_PATH)/$(LIBTBT_EXEC)
+
 clean:
 	-$(RM) $(LIBTBT_EXEC) $(O_FILES)
 
-.PHONY: all clean
+.PHONY: all clean install uninstall

